doc/classes/Vector2.xml

```diff
@@ -66,6 +66,8 @@
 			<description>
 				Returns this vector's angle with respect to the positive X axis, or [code](1, 0)[/code] vector, in radians.
 				For example, [code]Vector2.RIGHT.angle()[/code] will return zero, [code]Vector2.DOWN.angle()[/code] will return [code]PI / 2[/code] (a quarter turn, or 90 degrees), and [code]Vector2(1, -1).angle()[/code] will return [code]-PI / 4[/code] (a negative eighth turn, or -45 degrees).
+				The following graph illustrates what angle is returned:
+				[img]https://raw.githubusercontent.com/godotengine/godot-docs/master/img/vector2_angle.png[/img]
```

Comment:
I don't remember if `[img]` actually works in the editor and/or the online docs, but wouldn't it be too much to embed the same graph three times in the docs? An inline link to the graph or the documentation section which includes it might be better?

CC @Calinou @aaronfranke

Reply:
@akien-mga This PR adds links to 3 different graphs. I think they're helpful and worth adding, assuming `[img]` works.

Reply:
Indeed, the CI checks fail too:
```
ERROR: Unresolved type 'img', file: Vector2
ERROR: Unresolved type '/img', file: Vector2
```

Reply:
In hindsight, I should've paid attention to the [class reference writing guidelines](https://docs.godotengine.org/en/latest/community/contributing/class_reference_writing_guidelines.html) whether `[img]` is compatible, but on the other hand, the guidelines make no mention of `[url]` either.